var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -112,8 +112,9 @@ class Openblas(MakefilePackage):
     patch('openblas_fujitsu_v0.3.11.patch', when='@0.3.11: %fj')
     patch('openblas_fujitsu2.patch', when='@0.3.10: %fj')
 
-    # See https://github.com/spack/spack/issues/19932
-    conflicts('%gcc@:8.2.99', when='@0.3.11:')
+    # See https://github.com/spack/spack/issues/19932#issuecomment-733452619
+    conflicts('%gcc@7.0.0:7.3.99', when='@0.3.11:')
+    conflicts('%gcc@8.0.0:8.2.99', when='@0.3.11:')
```

Comment:
You can use a comma to separate these:
```suggestion
    conflicts('%gcc@7.0.0:7.3.99,8.0.0:8.2.99', when='@0.3.11:')
```

Reply:
Ah, of course! Thanks!